File: PAG_DTO/VM_PAG_LISTA_VALORES_DTO.cs
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;

namespace PAG_DTO
{
    public class VM_PAG_LISTA_VALORES_DTO
    {
        [DisplayName("Pk")]
        public decimal LLAVE_VIEW { get; set; }
        [DisplayName("Id Columna")]//MaxLength(4), ReadOnly(true)]
        public short ID_COLUMNA { get; set; }
        [DisplayName("Descripción")]// MaxLength(200), ReadOnly(true)]
        public string DESC_COLUMNA { get; set; }
        [DisplayName("Tabla")]//, MaxLength(30), ReadOnly(true)]
        public string TABLA { get; set; }
        [DisplayName("columna")]//, MaxLength(30), ReadOnly(true)]
        public string COLUMNA { get; set; } 
        [DisplayName("Tipo")]//, MaxLength(10), ReadOnly(true)]
        public string TIPO_VALOR { get; set; }
        [DisplayName("valor")]//, MaxLength(40), ReadOnly(true)]
        public string VALOR { get; set; }
        [DisplayName("Descripción")]//, MaxLength(200), ReadOnly(true)]
        public string DESCRIPCION { get; set; }
        [DisplayName("Otros Valores")]//], MaxLength(200), ReadOnly(true)]
        public string OTROS_VALORES { get; set; }

    }
}

File: PAG/app/Clases/VM_PAG_LISTA_VALORES_DTO.ts



    // $Classes/Enums/Interfaces(filter)[template][separator]
    // filter (optional): Matches the name or full name of the current item. * = match any, wrap in [] to match attributes or prefix with : to match interfaces or base classes.
    // template: The template to repeat for each matched item
    // separator (optional): A separator template that is placed between all templates e.g. $Properties[public $name: $Type][, ]

    // More info: http://frhagn.github.io/Typewriter/

    
    export class VM_PAG_LISTA_VALORES_DTO {
        
        // LLAVE_VIEW
        public LLAVE_VIEW: number = 0;
        // ID_COLUMNA
        public ID_COLUMNA: number = 0;
        // DESC_COLUMNA
        public DESC_COLUMNA: string = null;
        // TABLA
        public TABLA: string = null;
        // COLUMNA
        public COLUMNA: string = null;
        // TIPO_VALOR
        public TIPO_VALOR: string = null;
        // VALOR
        public VALOR: string = null;
        // DESCRIPCION
        public DESCRIPCION: string = null;
        // OTROS_VALORES
        public OTROS_VALORES: string = null;
    }
